identity-apps-core/apps/authentication-portal/src/main/webapp/user-not-found.jsp

```diff
@@ -124,23 +126,7 @@
         }
     }
 
-    try {
```

Comment:
This logic will be imported from the `registration-portal-url-resolver.jsp` page

https://github.com/wso2/identity-apps/blob/cb40a149208e7253456e5044096da2b5c9f7888c/identity-apps-core/apps/authentication-portal/src/main/webapp/util/registration-portal-url-resolver.jsp#L85-L96